### PR TITLE
Add support for removing enterprise Actions runners

### PR DESCRIPTION
**Pull request by zetaab**
adds support for

https://docs.github.com/en/rest/reference/enterprise-admin#delete-a-self-hosted-runner-from-an-enterprise

### CODE REVIEW

**Comment by zetaab** [comment]
@googlebot I signed it!

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:223]: I'm confused.  It looks to me like we already support this endpoint:

https://github.com/google/go-github/blob/master/github/enterprise_actions_runners.go#L17

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:247]: We support this one too:

https://github.com/google/go-github/blob/master/github/enterprise_actions_runners.go#L37

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:368]: This one appears to be new, but it should be moved alongside the other two endpoints listed above.

**Review by gmlewis — CHANGES_REQUESTED** [review]

File: github/actions_runners.go
Comment (zetaab) [reply github/actions_runners.go:247]: oh sorry I did not check that there are another file.. :)

File: github/actions_runners.go
Comment (zetaab) [reply github/actions_runners.go]: done

**Review by gmlewis — APPROVED** [review]
Thank you, @zetaab !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by zetaab** [comment]
@gmlewis now we could merge this?

**Review by gmlewis — APPROVED** [review]
Ah, thank you for the naming changes, @zetaab ... I'm glad you caught that, as I had missed it.
Thank you, @wesleimp for the code review.
LGTM.
Merging.

Oh, and in the future, @zetaab , we prefer that you don't force push to PRs in this repo because it typically makes the code reviews more challenging to see what was already reviewed... but in this case, it made no difference, fortunately... just an FYI for future contributions.  Thanks again!

**Pull request merged**
